downloads/platform.md

```diff
@@ -46,9 +46,9 @@ Julia installs all its files in a single directory. Deleting the directory where
 
 ## Windows
 
-Julia is available for Windows 7 and later for both 32 bit and 64 bit versions.
+Julia works best on 64-bit, and latest Windows 11 (you can expect terminal problems on default Windows 10), and is available for Windows 7 and later for both 32- and 64-bit versions.
```

Comment:
Please don't "sneak in" unrelated changes into a PR does a single thing (updates a link).